Update cdn

smalls opened the pull request:
In progress; as the last commit notes, this isn't working yet.

smalls committed:
update arcslib to PolymerLabs/arcs@f6e2c85

smalls committed:
copy arcs artifacts from PolymerLabs/arcs@f411dd8
dev> gulp && gulp copy-support

smalls committed:
in-progress update of ProductMultiplexer from PolymerLabs/arcs@f411dd8
Remove the no-longer-needed ProductMultipler2 source files (it now uses
a single source file and is declared separately in
ProductMultipler.manifest).
Note that this is not currently working; it fails with an error:

  Error: {Variable, [object Object]} is not of type Entity

smalls commented on dev/artifacts/Products/Products.recipes:
```diff
@@ -59,17 +58,17 @@ recipe
     person <- view1
   ProductMultiplexer
-    products <- view2
+    list <- view2
```
I should have mentioned this, I'm not sure which is preferred (`products` or `list`) but in this change I was thinking that it was easier to keep arcs & arcs-cdn in sync if they used the same identifier.

`products` is more descriptive, though. Should I change this and the version in arcs to that?

sjmiles replied:
Fwiw, my hope is that soonish the Multplexers are generic, in which case the input parameter will need a generic name, like `list`.

smalls replied:
Oh yes! Thanks makes total sense. And there's the start to that in ProductMultiplexer2 (one of the parameters is generic).

smalls committed:
change to concrete Product type (instead of the type variable)

smalls committed:
small fixes
- include TransformationDomParticle
- fix source paths
- rebuild (from the same arcs source)

smalls committed:
change back to generic type (undo b133a5eca76b46377a7e6134c8d8485e2c3f72ec)

smalls committed:
small tweaks to suggestion text

mariakleiner commented on dev/artifacts/Products/ProductMultiplexer.manifest:
```diff
@@ -1,19 +1,31 @@
+
+// TODO: rename to a generic (non product specific) multiplexer.
+particle ProductMultiplexer2 in 'source/ProductMultiplexer.js'
+  ProductMultiplexer2(host HostedParticleShape hostedParticle, in [~a] list, in [Product] others)
```
Please, change others to [~a] too

smalls committed:
more generics

smalls committed:
allow content.model to be undefined
